admin-console/src/main/webapp/src/pages/BundlePage.tsx

```diff
@@ -114,17 +155,20 @@ export const BundlePage: React.FunctionComponent = () => {
                                     <Button variant="link" component={ (props: any) =>
                                         <Link { ...props } to={ linkTo.application(a.id) } /> }>{ a.displayName }</Button>
                                 </Td>
-                                <Td>{ a.id }</Td>
+                                <Td>{ a.name}</Td>
                                 <Td>
                                     <ListEventTypes
                                         appId={ a.id }
                                     />
                                 </Td>
+                                <Td>{ a.id }</Td>
                                 <Td>
                                     <Button className='edit' type='button' variant='plain'
+                                        onClick={ () => editApplication(a) }
                                     > { <PencilAltIcon /> } </Button></Td>
                                 <Td>
                                     <Button className='delete' type='button' variant='plain'
+                                        onClick={ () => deleteApplicationModal(a) }
                                     >{ <TrashIcon /> } </Button></Td>
```

Comment:
I think we could group the actions in a kebab: https://patternfly-react.surge.sh/components/table#composable-actions